lib/Builder/SourceFileBuilder.php

```diff
@@ -18,31 +19,42 @@ class SourceFileBuilder
     /** @var Parsedown */
     private $parsedown;
 
+    /** @var RSTParser */
+    private $rstParser;
+
     public function __construct(
         SourceFileRenderer $sourceFileRenderer,
         Filesystem $filesystem,
-        Parsedown $parsedown
+        Parsedown $parsedown,
+        RSTParser $rstParser
     ) {
         $this->sourceFileRenderer = $sourceFileRenderer;
         $this->filesystem         = $filesystem;
         $this->parsedown          = $parsedown;
+        $this->rstParser          = $rstParser;
     }
 
     public function buildFile(SourceFile $sourceFile, string $buildDir) : void
     {
-        $rendered = $sourceFile->getContents();
+        $contents = $sourceFile->getContents();
 
         if ($sourceFile->isMarkdown()) {
```

Comment:
Move this else/else/else thing to a `private` method with early return expressions, please :)